pulsarlib/api.go

```diff
@@ -480,4 +481,27 @@ func CreateSubscriptionOnTopic(tenantID, nameSpace, topicName, subscriptionName,
 		return fmt.Errorf("Error in creating subscription, Http Response [%v]", resp.StatusCode)
 	}
 	return nil
-}
\ No newline at end of file
+}
+
+func DeleteSubscriptionOnTopic(tenantID, nameSpace, topicName, subscriptionName string) error {
```

Comment:
namespace is a single word. remove camelcase from it. Remove CreateSubscriptionOnTopic function as well